platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_structuring_builds.adoc

```diff
@@ -498,18 +498,32 @@ When using a hierarchical directory structure to organize your Gradle projects,
 When you use the link:{kotlinDslPath}/gradle/org.gradle.api.initialization/-settings/include.html[Settings.include()] method to include a project in your Grade settings file, you typically include projects by supplying the directory name like `include("featureA")`.
 This usage assumes that `featureA` is located at the root of your build.
 
-You can also include projects located in subdirectories using hierarchical paths with the `:` charactor as a separator.
-This allows you to organize large builds containing many projects to improve comprehensibility, versus having all your projects live at the root of the build.
+You can also include projects located in deeper subdirectories by specifying the complete path to the project using the `:` charactor as a path separator.
 For instance, if project `search` was located in a subdirectory named `features`, itself located in a subdirectory named `subs`, you could call `include(":subs:features:search")` to include it.
 
-However, without further configuration, Gradle will create empty projects for every element of the path.
+Nesting projects under a sensible hierarchical directory structure is a common practice in larger Gradle builds including more than a dozen or so projects.
+Such a layout allows you to organize large builds and improve comprehensibility, versus having all your projects live at the root of the build.
+
+However, without further configuration, Gradle will create empty projects for every element of a hierarchical path.
 In the example above, Gradle will create a project named `:subs`, a project named `:subs:features`, and a project named `:subs:features:search`.
-This probably isn't intended, as you likely only want to include the `search` project.
-Additional projects - even if empty - will slow down your build, clutter reporting, and make your build harder to understand.
+This probably isn't intended, as you likely only want to add a `search` project.
+
+Unused projects - even if empty - can surprise maintainers, clutter reporting, and make your build harder to understand.
+If plugins are applied using `allprojects { ... }` or `subprojects { ... }`, they will apply to each of these empty projects, which can also have performance impacts.
```

Comment:
```suggestion
They also introduce unintended side effects.
If you use `allprojects { ... }` or `subprojects { ... }`, plugins and configuration blocks will apply to every project, including the empty ones. 
This can degrade build performance.
```